test/unit/cli/webroot-locator-test.js

```diff
@@ -0,0 +1,73 @@
+var proxyquire = require('proxyquire').noCallThru()
+var simple = require('simple-mock')
+var test = require('tap').test
+var path = require('path')
+
+var fsMock = {
+  'existsSync': simple.spy(),
+  'statSync': simple.spy()
+}
+
+var statsMock = {
+  'isDirectory': simple.spy()
+}
+
+test('webroot locator', function (group) {
+  var webrootLocator = proxyquire('../../../cli/webroot-locator',
+    {fs: fsMock})
+
+  group.afterEach(function (done) {
+    fsMock.existsSync.actions.length = 0
```

Comment:
okay I understand what `stub.actions` are now. I didn’t know one can chain `callbackWith`, `returnWith` and `throwWith`, that’s pretty cool! I would think that `stub.reset()` would remove these though, I’ll file an issue on simple-mock.

But please change the `actions.length = 0` to `actions = []`, it’s shorter and more readable